src/core/js/helpers.js

```diff
@@ -101,12 +101,47 @@ define([
 
         /**
          * makes the _globals object in course.json available to a template
-         */ 
+         */
         import_globals: function(context) {
-            if(!context.data.root._globals) {
-                context.data.root._globals = Adapt.course.get('_globals');
+            if (context.data.root._globals) return "";
+            context.data.root._globals = Adapt.course.get('_globals');
+            return "";
+        },
+
+        /**
+         * makes the Adapt module data available to a template
+         */
+        import_adapt: function(context) {
+
+            if (context.data.root.Adapt) return;
+            var adapt = context.data.root.Adapt = {};
```

Comment:
P.S. this one-liner took me a while to read.

Reply:
wow you're a slow reader ;-)